packages/gestalt-cli/src/index.js

```diff
@@ -30,9 +30,11 @@ commander
     'update schema.json and create a database migration based on changes to ' +
     'schema.graphql in a Gestalt project directory.'
   )
-  .option('-u', '--url <url>', 'the url to your local database')
-  .action(async () => {
-    await migrate();
+  .option('-d, --directory <path>', 'the directory of your mutation files')
+  .option('-g, --glob <pattern>', 'a glob pattern for selecting mutation files')
```

Comment:
What do you think about naming this `--mutations-glob` and `--mutations-directory`?  I'm just thinking if you are running `migrate`, its not totally clear that 'directory' would refer to mutations.